src/components/App.js

```diff
@@ -1,77 +0,0 @@
-import React, { Component } from 'react';
-import {connect} from 'react-redux'
-import {bindActionCreators} from 'redux'
-
-import {checkLocation, echoWebSocket, testLatency, getWeather} from '../actions/main'
-
-import WeatherCanvas from './WeatherCanvas'
-
-import css from '../style/main.less'
-
-class App extends Component {
-
-    constructor(props){
-        super(props)
```

Comment:
Like in Java OOP. 
In react case, we always need to invoke the parent constructor in order to instantiate the component with the current props argument.